.gitignore

```diff
@@ -87,3 +87,5 @@ release.json
 messages.mo
 
 docker/requirements-local.txt
+
+superset/templates/custom_override_template.html
```

Comment:
The idea with adding the gitignore entry was so that 
1. There is an example file
2. The file is tracked, but future changes to the file are not. This allows users to add changes to the file without needing to commit those changes or manage that diff.
3. Changes can be committed via `git add -f superset/templates/custom_override_template.html` if necessary. 

Though I see how this is could likely become source of confusion for those unfamiliar with this pattern. I'll remove the gitignore entry.

How does `tails_js_custom_extra.html` sound? To me`tai_js_extra.html` sounds like an internal name and not really like a vector for customization.

Reply:
```
The file is tracked, but future changes to the file are not.
```

This is not true. Once a file is tracked, any changes you made would show up in `git status` and will be committed if you do `git commit -a` (even without `-f`).

`tail_js_custom_extra` sounds good.

Reply:
Ah, you are correct. You can ignore entire directories, except for one file, by adding the dir to `.gitignore` then adding the file with with `git add -f ...`. I assumed it worked similarly for files and subsequent changes 😅